consensus/src/event_processor.rs

```diff
@@ -722,6 +726,7 @@ impl<T: Payload> EventProcessor<T> {
                 return;
             }
         } else {
+            // REVIEW: why?
```

Comment:
I asked the same question lol, see https://toroeng.slack.com/archives/CH9HSH0C9/p1584488500068600